osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -61,8 +61,8 @@ def on_moved(self, event):
                     # This means it was put into a place on the hierarchy being watched but otherwise not attached to a
                     # node, so it needs to be added just like a new event rather than as a move.
 
-                    new_event = DirCreatedEvent(event.dest_path) if event.is_directory else FileCreatedEvent(event.dest_path)
-                    yield from self._create_file_or_folder(new_event, src_path=dest_path)
+                    # new_event = DirCreatedEvent(event.dest_path) if event.is_directory else FileCreatedEvent(event.dest_path)
+                    # yield from self._create_file_or_folder(new_event, src_path=dest_path)
```

Comment:
It was one of the things that broke in 5129. After the node category issue is fixed, this code ends up running and it creates a duplicate of the file/folder. However, I am NOT currently getting an error right now with that code. It is being run, but is not breaking anything. Thus, I have uncommented it.